### ISSUE TITLE
question

### ISSUE THREAD

**Issue by 2606004789** [yjzscode/Optimal-Normalisation-in-Color-Spaces#2]
Can pre-training parameters be provided for testing？

model = SA3(model, self.device, mu0_lab=mu0_lab, sigma0_lab=sigma0_lab, mu0_hsv=mu0_hsv,sigma0_hsv=sigma0_hsv, mu0_rgb=mu0_rgb,sigma0_rgb=sigma0_rgb)
        state_dict = model.load_state_dict(torch.load(
            "./staintrick/results/20230206_01:44_HsvTem_1_vgg11/weights/best_model.pth",
            map_location='cpu', strict=False))
